tests/robot-tests/tests/general_public/statistics_page.robot

```diff
@@ -4,18 +4,164 @@ Resource            ../libs/public-common.robot
+
+Open and validate "Pupils and schools" accordion section (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user opens accordion section    Pupils and schools
+
+    user waits until accordion section contains text    Pupils and schools    Academy transfers
+
+    user waits until accordion section contains text    Pupils and schools    Admission appeals
+
+    user waits until accordion section contains text    Pupils and schools    Exclusions
+
+    user waits until accordion section contains text    Pupils and schools    Parental responsibility measures
+
+    user waits until accordion section contains text    Pupils and schools    Pupil absence
+
+    user waits until accordion section contains text    Pupils and schools    Pupil projections
+
+    user waits until accordion section contains text    Pupils and schools    School and pupil numbers
 
-Validate accordion sections exist
+    user waits until accordion section contains text    Pupils and schools    School applications
+
+    user waits until accordion section contains text    Pupils and schools    School capacity
+
+    user waits until accordion section contains text    Pupils and schools    Special educational needs (SEN)
+
+Open and validate "School and college outcomes and performance" accordion section (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user opens accordion section    School and college outcomes and performance
+
+    user waits until accordion section contains text    School and college outcomes and performance
+    ...    16 to 19 attainment
+
+    user waits until accordion section contains text    School and college outcomes and performance
+    ...    GCSEs (key stage 4)
+
+    user waits until accordion section contains text    School and college outcomes and performance    Key stage 1
+
+    user waits until accordion section contains text    School and college outcomes and performance    Key stage 2
+
+Open and validate "Teachers and school workforce" accordion section (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user opens accordion section    Teachers and school workforce
+
+    user waits until accordion section contains text    Teachers and school workforce    Initial teacher training (ITT)
+
+    user waits until accordion section contains text    Teachers and school workforce    School workforce
+
+Open and validate "UK education and training statistics" accordion section (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user opens accordion section    UK education and training statistics
+    user waits until accordion section contains text    UK education and training statistics
+    ...    UK education and training statistics
+
+# below onwards assertions targets local and dev environments
+
+Navigate to Find Statistics page (local, dev)
```

Comment:
You could have this test case run against all environments, rather than repeating it twice.

My general thought for how this file might be is something like this:

```
Force Tags Prod
*** Test Cases ***
Navigate to Find stats page
[Tags]  Local Dev   # Prod tag is applied by the Force Tags so no need to add it here
....
Check bootstrapped data
[Tags] Local Dev NotAgainstProd
...
First test case to run against prod
# no tags required
...
```
etc.

I would personally veer towards having one test case that does a quick sanity check against Local / Dev - I think it could just check the bootstrapped data, which shouldn't change - if it is too thorough then changing test data will cause failures moving forward.